### DISCUSSION TITLE
キャラメイク時に再生するBGMを設定できるようにしてほしい

### DISCUSSION THREAD

**Discussion by chikatei** [hengband/hengband#773] [Closed] [answered]
変愚蛮怒複合パッチ適用版や東方勝手版のように、本家変愚でもキャラメイク時に再生するBGMを設定できるようにしてはどうでしょうか。種族等の選択に加えて生い立ち記述などそれなりに時間がかかる場面なので、無音でなくBGMが再生できると雰囲気が出て楽しいと思います。

**Comment by shimitei** [comment]
キャラメイク時のBGMに対応する予定です。
Alpha17でモンスター遭遇BGM機能のリリースができたので、
BGM処理まわりのリファクタリングついでに対応するつもりでした。

余談ですが、実は現バージョンでもキャラメイク時にBGM再生の処理が行われているのです。
しかしながらBGMフラグ関連の初期化処理がキャラメイク後だったりで再生されることがないという。

**Reply by chikatei** [reply]
対応予定でしたか、ありがとうございます。
楽しみにしております！

**Comment by iks3** [comment, marked as answer]
対応されましたのでこのスレッドは終了いたします。
